src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/WrapWithTagTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.VisualStudio.Razor.IntegrationTests;
+
+public class WrapWithTagTests : AbstractRazorEditorTest
+{
+    [IdeFact]
+    public async Task WrapWithTag_RootLevelElement()
+    {
+        // Open the file
+        await TestServices.SolutionExplorer.OpenFileAsync(RazorProjectConstants.BlazorProjectName, RazorProjectConstants.CounterRazorFile, ControlledHangMitigatingCancellationToken);
+
+        await TestServices.Editor.PlaceCaretAsync("h1", charsOffset: -1, ControlledHangMitigatingCancellationToken);
+
+        await TestServices.Editor.WaitForComponentClassificationAsync(ControlledHangMitigatingCancellationToken);
+
+        // Act
+        TestServices.Input.Send("%+w");
```

Comment:
I assume this is `Alt+Shift+w`? I prefer doing it this way over using commands, but we might want to note what the string means since it's a bit arcane.